packages/framework/client-logger/fluid-telemetry/src/container/telemetryProducer.ts

```diff
@@ -50,10 +50,12 @@ export class ContainerEventTelemetryProducer {
 		}
 	}
 
-	public produceHeartbeatTelemetry = (): ContainerHeartbeatTelemetry => {
-		return this.produceBaseContainerTelemetry(
-			ContainerTelemetryEventNames.HEARTBEAT,
-		) as ContainerHeartbeatTelemetry;
+	public produceHeartbeatTelemetry = (): IFluidTelemetry => {
+		return {
+			eventName: "fluidframework.container.heartbeat",
+			containerId: this.containerId,
+			containerInstanceId: this.containerInstanceId,
+		} as unknown as IFluidTelemetry;
```

Comment:
Hmm, need the casts because this event name is not part of the ones in the type definition... will think more about this, but let's keep this for now.